backend/corpora/dbnl/dbnl.py

```diff
@@ -261,18 +262,18 @@ def _xml_files(self):
             Pass(
                 Backup(
                     XML( # get the language on chapter-level if available
+                        CurrentTag(),
                         attribute='lang',
                         transform=lambda value: [value] if value else None,
                     ),
                     XML( # look for section-level codes
-                        {'name': 'div', 'attrs': {'type': 'section'}},
+                        Tag('div', type='section'),
```

Comment:
Can this search for a given attribute be passed as a flat keyword argument, or shouldn't it be `attrs={'type': 'section'}` instead?

Reply:
BeautifulSoup supports either. In some cases, `attrs` is preferred to prevent overlap with named arguments to `find_all()`.